src/transformation/visitors/return.ts

```diff
@@ -93,7 +93,8 @@ export function createReturnStatement(
 
     if (isInTryCatch(context)) {
         // Bubble up explicit return flag and check if we're inside a try/catch block
-        results.unshift(lua.createBooleanLiteral(true));
+        // results.unshift(lua.createBooleanLiteral(true));
+        results = [lua.createBooleanLiteral(true), ...results];
```

Comment:
I'm not sure this is really an improvement, commented code should be removed either way

Reply:
Is allocating a new array and copying really faster than unshift?